### ISSUE TITLE
Error in printing of cones

### ISSUE THREAD

**Issue by raulepure** [oscar-system/Polymake.jl#244]
I  construct a cone via rays given by a JULIA array, but then an error is thrown after construction, although the object itself is constructed and I can work with it.

I use JULIA 1.2 on Ubuntu 18.04

Here is the minimal example that produces an error:
julia> A = [1 2 3; 1 -1 0]

julia> c1 = @pm polytope.Cone(RAYS = A)
type: Cone<Rational>
Error showing value of type Polymake.BigObjectAllocated:
ERROR: MethodError: objects of type Regex are not callable
Stacktrace:
 [1] count(::Regex, ::SubString{String}) at ./reduce.jl:772
 [2] show(::IOContext{REPL.Terminals.TTYTerminal}, ::MIME{Symbol("text/plain")}, ::Polymake.BigObjectAllocated) at /home/epure/.julia/packages/Polymake/kIMyn/src/visual.jl:26
 [3] display(::REPL.REPLDisplay, ::MIME{Symbol("text/plain")}, ::Any) at /home/epure/julia/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:132
 [4] display(::REPL.REPLDisplay, ::Any) at /home/epure/julia/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:136
 [5] display(::Any) at ./multimedia.jl:323
 [6] #invokelatest#1 at ./essentials.jl:790 [inlined]
 [7] invokelatest at ./essentials.jl:789 [inlined]
 [8] print_response(::IO, ::Any, ::Bool, ::Bool, ::Any) at /home/epure/julia/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:156
 [9] print_response(::REPL.AbstractREPL, ::Any, ::Bool, ::Bool) at /home/epure/julia/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:141
 [10] (::getfield(REPL, Symbol("#do_respond#38")){Bool,getfield(REPL, Symbol("##48#57")){REPL.LineEditREPL,REPL.REPLHistoryProvider},REPL.LineEditREPL,REPL.LineEdit.Prompt})(::Any, ::Any, ::Any) at /home/epure/julia/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:718
 [11] #invokelatest#1 at ./essentials.jl:790 [inlined]
 [12] invokelatest at ./essentials.jl:789 [inlined]
 [13] run_interface(::REPL.Terminals.TextTerminal, ::REPL.LineEdit.ModalInterface, ::REPL.LineEdit.MIState) at /home/epure/julia/usr/share/julia/stdlib/v1.2/REPL/src/LineEdit.jl:2306
 [14] run_frontend(::REPL.LineEditREPL, ::REPL.REPLBackendRef) at /home/epure/julia/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:1038
 [15] run_repl(::REPL.AbstractREPL, ::Any) at /home/epure/julia/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:201
 [16] (::getfield(Base, Symbol("##737#739")){Bool,Bool,Bool,Bool})(::Module) at ./client.jl:390
 [17] #invokelatest#1 at ./essentials.jl:790 [inlined]
 [18] invokelatest at ./essentials.jl:789 [inlined]
 [19] run_main_repl(::Bool, ::Bool, ::Bool, ::Bool, ::Bool) at ./client.jl:374
 [20] exec_options(::Base.JLOptions) at ./client.jl:312
 [21] _start() at ./client.jl:464
  
julia> c1.RAYS
pm::Matrix<pm::Rational>
1 2 3
1 -1 0

**Comment by benlorenz** [comment]
Could you please try with Julia 1.3?
@kalmarek might have an idea as he recently reworked that code. But Julia 1.2 is not really supported anymore and we don't test this version.

**Comment by kalmarek** [comment]
yes, it's probably a lousy regular expression matching on my side;

@raulepure the error happens in code responsible for displaying the cone, If You just need to work with them you can ignore it for now.

**Comment by saschatimme** [comment]
There is a new method in Julia 1.3 which is called for `count(::Regex, ::AbstractString)`.
```julia
function count(t::Union{AbstractString,Regex}, s::AbstractString; overlap::Bool=false)
    n = 0
    i, e = firstindex(s), lastindex(s)
    while true
        r = findnext(t, s, i)
        isnothing(r) && break
        n += 1
        j = overlap || isempty(r) ? first(r) : last(r)
        j > e && break
        @inbounds i = nextind(s, j)
    end
    return n
end
```

Defining this should fix the problem in 1.2.

However, updating to Julia 1.3 fixes this problem also

**Comment by kalmarek** [comment]
this works fine on 1.4 as well
